openmc/deplete/chain.py

```diff
@@ -256,6 +304,10 @@ def from_endf(cls, decay_files, fpy_files, neutron_files, progress=True):
             List of ENDF neutron-induced fission product yield sub-library files
         neutron_files : list of str or openmc.data.endf.Evaluation
             List of ENDF neutron reaction sub-library files
+        reactions : iterable of str, optional
+            Transmutation reactions to include in the depletion chain, e.g.,
+            `["(n,2n)", "(n,gamma)"]`. Note that fission is always included if
+            it is present.
```

Comment:
Perhaps something could be added to this doc string so users are more aware of that useful list of reactions in chain.Reactions.keys()

Reply:
Good suggestion @Shimwell. I've just pushed a commit that adds documentation of `REACTIONS` and references it here.

Reply:
thanks for that